etc/kayobe/stackhpc-overcloud-dib.yml

```diff
@@ -0,0 +1,212 @@
+---
+# StackHPC overcloud host disk image configuration.
+
+###############################################################################
+# Diskimage-builder configuration for overcloud host disk images.
+
+# StackHPC overcloud host disk image Diskimage Builder recipe. This may be used
+# as an item in the overcloud_dib_host_images list when
+# overcloud_dib_build_host_images is true.
+# Example:
+# overcloud_dib_host_images:
+#   - "{{ stackhpc_overcloud_dib_host_image }}"
+stackhpc_overcloud_dib_host_image:
```

Comment:
Wondering if this name should be any more specific than this? Will this be the only image we provide?

Reply:
I guess let's start with that and see if we need more (hope not).
Do we plan to build those images in some CI and upload to Ark? Would help environments without ,,normal'' internet access.

Reply:
Yes. I have a workflow to build the images in progress, but not uploading to Ark yet. Probably @jackhodgkiss will look at that.